daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Inputs.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.transaction
+
+import com.daml.lf.data.Ref
+import com.daml.lf.value.Value.ContractId
+
+import scala.collection.immutable.TreeSet
+
+/** The inputs of the transaction, that is:
+  *   - the contracts were fetched
+  *   - all the parties involved
+  *   - all the contract keys involved.
+  */
+/** @param contracts
+  *     The contracts fetched by this transaction.
+  *
+  * @param parties
+  *     The contracts created by this transaction.
```

Comment:
This line doesn't fit with the `parties` parameter name. This comment looks like a copy-n-paste error.